src/javascript/app/pages/form/form-component.jsx

```diff
@@ -0,0 +1,421 @@
+import React, { useEffect, useState } from 'react';
+import {
+    TextField,
+    TextFieldAddon,
+    DatePickerDropdown,
+    Checkbox,
+    SectionMessage,
+    BreakpointProvider,
+} from '@deriv-com/quill-ui';
+import moment from 'moment';
+import { CurrencyDropdown } from './currency-dropdown.jsx';
+import { NumbersDropdown } from './numbers-dropdown.jsx';
+import BarrierFields from './barrier-fields.jsx';
+import { TimePickerDropdown } from './time-selection.jsx';
+import { DropdownComponent } from './dropdown-component.jsx';
+import Defaults, { PARAM_NAMES } from '../trade/defaults.js';
+import {
+    useSessionChange,
+    useTradeChange,
+    eventDispatcher,
+} from '../../hooks/events.js';
+import common_functions from '../../../_common/common_functions.js';
+import { localize } from '../../../_common/localize.js';
+import dataManager from '../../common/data_manager.js';
+
+export const FormComponent = () => {
+    const [tradeData, setTradeData] = useState({});
+
+    const hasTradeChange = useTradeChange();
+    const hasSessionChange = useSessionChange();
+
+    useEffect(() => {
+        setTradeData((oldData) => ({
+            ...oldData,
+            ...dataManager.getAllTrades(),
+        }));
+    }, [hasTradeChange, hasSessionChange]);
+
+    const formName = Defaults.get(PARAM_NAMES.FORM_NAME);
+    const expiryType = Defaults.get(PARAM_NAMES.EXPIRY_TYPE);
```

Comment:
```suggestion
    const form_name = Defaults.get(PARAM_NAMES.FORM_NAME);
    const expiry_type = Defaults.get(PARAM_NAMES.EXPIRY_TYPE);
```
in the same fashion as the ones below :)